design-documents/graph-ql/coverage/company.md

```diff
@@ -174,7 +174,18 @@ type DeleteCompanyTeamOutput @doc(description: "Delete company team output data
 }
 
 type CreateCompanyOutput @doc(description: "Create company output data schema.") {
-    company: Company! @doc(description: "New company instance.")
+    company: CompanyOutput! @doc(description: "Create company output instance.")
+}
+
+type CompanyOutput {
```

Comment:
Propose to include these fields under `CreateCompanyOutput` and not have `CompanyOutput`. Also, let's rename the following fields for consistency and copy description for the fields from `CompanyCreateInput`.
`name` to `company_name`
`email` to `company_email`
`vat_id` to `vat_tax_id`